x/concentrated-liquidity/tick_test.go

```diff
@@ -109,6 +110,7 @@ func (s *KeeperTestSuite) TestInitOrUpdateTick() {
 			tickExists:             false,
 			expectedLiquidityNet:   DefaultLiquidityAmt.Neg(),
 			expectedLiquidityGross: DefaultLiquidityAmt,
+			minimumGasConsumed:     uint64(10000),
```

Comment:
Building up on my previous comment, consider importing this from `tick.go`. That way, if we end up changing that value later, we only need to update one constant and zero tests